frontend/src/components/EditableCell.jsx

```diff
@@ -60,7 +60,7 @@ const EditableCell = ({
       if (typeof record.value === "string") {
         setLocationName(record.value);
       } else {
-        config.fn.administration(record.value, false).then((res) => {
+        config.fn.administration(record.id, false).then((res) => {
```

Comment:
Please revert this change, because record. id means:  `Question. id` and it will take us to the wrong response.

eg:

[http://localhost:3000/api/v1/administration/1699354849382](http://localhost:3000/api/v1/administration/1699354849382)